CMakeLists.txt

```diff
@@ -52,7 +52,7 @@ endif()
 option( DETRAY_EIGEN_PLUGIN "Build Eigen math plugin" ON )
 option( DETRAY_SMATRIX_PLUGIN "Build ROOT/SMatrix math plugin" OFF )
 option( DETRAY_VC_PLUGIN "Build Vc based math plugin" ON )
-option( DETRAY_SVG_DISPLAY "Build ActSVG display module" OFF)
+option( DETRAY_SVG_DISPLAY "Build ActSVG display module" ON)
```

Comment:
The testing depends on the svg output by now, so this should probably be a dependent option

Reply:
Should I set it to something like `option( DETRAY_SVG_DISPLAY "Build ActSVG display module" ${DETRAY_BUILD_TESTING})`?